ui/src/Components/SearchCandidates.jsx

```diff
@@ -112,24 +122,21 @@ export default class SearchCandidates extends React.Component {
 						<FormGroup row>
 							<Label for="text-search" sm={2}>Name</Label>
 							<Col sm={10}>
-							<SearchBox onSearchChange={this.onNameSearch} />
+							<input type="text" className="input" onChange={this.onNameSearch} placeholder="John Doe"/>
 							</Col>
 						</FormGroup>
 						<FormGroup row>
 							<Label for="email" sm={2}>Email</Label>
 							<Col sm={10}>
-							<SearchBox onEmailSearch={this.onEmailSearch} />
+							<input type="text" className="input" onChange={this.onEmailSearch} placeholder="abc@gmail.com"/>
 							</Col>
 						</FormGroup>
 						<FormGroup row>
 							<Label for="phone" sm={2}>Phone Number</Label>
 							<Col sm={10}>
-							<SearchBox onNumberSearch={this.onNumberSearch} />
+							<input type="text" className="input" onChange={this.onNumberSearch} placeholder="6471231234"/>
 							</Col>
 						</FormGroup>
-						<FormGroup check row>
-							<Button>Search</Button>
```

Comment:
I saw your comment that you removed the search button because it's not needed to apply the search. However, I think we could still add value to the users by having that search button just clear the search criteria (and still display the results). Just some food for thought :)